compiler-plugin-test/src/test/resources/kotlin/ItExtAlignForExampleAggregate.kt

```diff
@@ -0,0 +1,16 @@
+import it.unibo.collektive.aggregate.api.Aggregate
+import it.unibo.collektive.Collektive.Companion.aggregate
+
+fun Aggregate<Int>.exampleAggregate() {}
+
+// ========= ========= ========= ========= ========= ========= ========= ========= ========= =========
+// END OF BOILERPLATE CODE
+// ========= ========= ========= ========= ========= ========= ========= ========= ========= =========
+
+fun Aggregate<Int>.entry() {
+    alignedOn(0) {
+        for(i in 1..3) {
+            exampleAggregate()
```

Comment:
```suggestion
            neighboring(1)
```

Simpler, and allows you to remove lines 6--8. Do something similar for the other tests, too

Reply:
Ok, so I will remove `exampleAggregate` and just use standard aggregate functions.